Sources/PrimerSDK/Classes/Core/Checkout Components/PrimerHeadlessUniversalCheckoutUIManager.swift

```diff
@@ -878,7 +878,8 @@ extension PrimerHeadlessUniversalCheckout.CardFormUIManager {
 
 extension PrimerHeadlessUniversalCheckout.CardFormUIManager: SFSafariViewControllerDelegate {
     
-    func safariViewControllerDidFinish(_ controller: SFSafariViewController) {
+    public func safariViewControllerDidFinish(_ controller: SFSafariViewController) {
```

Comment:
Why?

Reply:
I was getting an error from the compiler:

```
func safariViewControllerDidFinish(_ controller: SFSafariViewController) nearly matches optional requirement 'func safariViewControllerDidFinish(_ controller: SFSafariViewController)
```

Can revert. I guess is due to the fact that the class is `public` ?

Reply:
I didn't have any problems building, was there a problem when you archived?